package.json

```diff
@@ -44,7 +44,12 @@
     "@angular/router": "4.3.6",
     "@blackbaud/auth-client": "2.2.0",
     "@blackbaud/skyux-lib-help": "1.1.0",
+    "@blackbaud/skyux-logger": "1.0.0",
```

Comment:
I'm assuming we'll need to update this version once your patch gets in. Happy to approve the pull request once it gets released!